test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -142,14 +142,14 @@ proc ::AlgorandGoal::StartNetwork { NETWORK_NAME NETWORK_TEMPLATE TEST_ALGO_DIR
 
     # Running on ARM64, it seems that network creation is pretty slow.
     # 30 second won't be enough here, so I'm changing this to 90 seconds.
```

Comment:
Update comment. If you know why 90 seconds is no longer enough maybe add that in too.

Reply:
Updated comment.